holoviews/tests/plotting/bokeh/test_elementplot.py

```diff
@@ -822,6 +823,21 @@ def test_element_backend_opts_model_not_resolved(self):
             "WARNING", "cb model could not be"
         )
 
+    def test_dynamicmap_subcoordinate_y_enabled_labels_mismatched(self):
+        def func(data):
+            if not data:
+                return Curve([]) * Curve([])
```

Comment:
What happens if this is something like this:

- `Curve([], label="1") * Curve([])`
- `Curve([], label="3") * Curve([], label="4")`